app.js

```diff
@@ -6,20 +6,34 @@
 // a number of minutes as an argument and returns
 // the number of seconds
 
-function convertToSeconds() {}
+function convertToSeconds(minutes, seconds) {
+    seconds = minutes * 60;
+    return seconds;
+}
 
 console.log(`convertToSeconds(10) === 600 : ${convertToSeconds(10) === 600}`);
 
 // 2. areaOfTriangle
 // Write a function called areaOfTriangle that takes in the base and height
 // of a triangle and returns its area
-function areaOfTriangle() {}
+function areaOfTriangle(area, base, height) {
+    base = ( 5 );
+    height = ( 10 );
```

Comment:
This totally works for this specific instance of the problem, but something to be careful about going forward. You're resetting the value of base to 5 and height to 10. It wouldn't console log true if the values were different.